src/index.d.ts

```diff
@@ -1,115 +1,151 @@
 
   interface Core {
-    initCCP(containerDiv: HTMLElement, options: InitCCPOptions): void;
+    /**
+     * Integrates with Amazon Connect by loading the pre-built CCP located at `ccpUrl` into an iframe and placing it into the `containerDiv` provided.
+     * API requests are funneled through this CCP and agent and contact updates are published through it and made available to your JS client code.
+     *
+     * @param containerDiv The `<div>` DOM element to place the CCP.
+     * @param options The CCP init options.
+     */
+    initCCP(containerDiv: HTMLDivElement, options: InitCCPOptions): void;
+
+    /**
+     * Unitializes Amazon Connect Streams. Removing any subscription methods that have been called.
+     * The CCP iframe will not be removed though, so you'll have to manually remove it.
+     */
+    terminate(): void;
   }
-  let core: Core;
+
+  const core: Core;
 
   interface SoftPhoneOptions {
-    /*
-    * Whether to disable the ringtone.
-    */
-    disableRingtone?: boolean;
-    /*
-    * Whether to display the softphone in a frame.
-    */
-    allowFramedSoftphone?: boolean;
-    /*
-    * A URL for a custom ringtone.
-    */
-    ringtoneUrl?: string;
+    /**
+     * Normally, the softphone microphone and speaker components are not allowed to be hosted in an iframe.
+     * This is because the softphone must be hosted in a single window or tab.
+     * The window hosting the softphone session must not be closed during the course of a softphone call or the call will be disconnected.
+     * If `allowFramedSoftphone` is `true`, the softphone components will be allowed to be hosted in this window or tab.
+     */
+    readonly allowFramedSoftphone?: boolean;
+
+    /** This option allows you to completely disable the built-in ringtone audio that is played when a call is incoming. */
+    readonly disableRingtone?: boolean;
+
+    /** If the ringtone is not disabled, this allows for overriding the ringtone with any browser-supported audio file accessible by the user. */
+    readonly ringtoneUrl?: string;
   }
 
   interface InitCCPOptions {
-    /*
-    * The URL for the Connect CCP.
-    */
-    ccpUrl: string;
-    /*
-    * The login URL for the Connect CCP when configured for SAML2 authentication.
-    */
-    loginUrl?: string;
-    /*
-    * Whether to display the login view.
-    */
-    loginPopup?: boolean;
-    /*
-    * Whether to auto-close the CCP popup window.
-    */
-    loginPopupAutoClose?: boolean;
-    /*
-    * Options specifying softphone configuration.
-    */
-    softphone?: SoftPhoneOptions;
+    /**
+     * The URL of the CCP.
+     * This is the page you would normally navigate to in order to use the CCP in a standalone page, it is different for each instance.
+     */
+    readonly ccpUrl: string;
+
+    /**
+     * Amazon connect instance region. Only required for chat channel.
+     * @example "us-west-2" // for PDX ccp instance.
```

Comment:
I don't think airport codes are public AWS vernacular, use `US West (Oregon)` instead?

Reply:
I thought so too. This was [taken from the current version of Documentation.md](https://github.com/amazon-connect/amazon-connect-streams/blob/master/Documentation.md#connectcoreinitccp).

Will update TypeScript and the `.md` files.